daemon/bindata.go

```diff
@@ -1,5 +1,9 @@
 // Code generated by go-bindata.
 // sources:
+// ../bpf/.bpf_lxc.c.swn
```

Comment:
Ah I just added it to .gitignore, how about a follow up patch to clean up GO_BINDATA? Its not really related to this patch series.